about.html

```diff
@@ -7,12 +7,137 @@
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@700&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;1,500;1,700&family=Poppins:ital,wght@0,500;0,600;0,700;1,200;1,400;1,500;1,700&family=Roboto:ital,wght@0,100;0,300;0,400;1,100;1,300;1,400&family=Space+Mono&display=swap" rel="stylesheet">
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
   <link rel="stylesheet" href="about.css">
 </head>
 <body>
-  <p>a</p>
+  <header>
+    <div class="container-fluid header-bg position-fixed">
+      <div class="d-flex justify-content-end">
+        <div class="d-flex justify-content-between align-items-center gap-3 me-5">
+          <a href="#/"><i class="fa fa-facebook white pb-2"></i></a>
+          <a href="#/"><i class="fa fa-twitter white pb-2"></i></a>
+          <span class="white header-text">English</span>
+          <span class="white header-text">My Page</span>
+          <span class="white header-text">Logout</span>
+        </div>
+      </div>
+    </div>
+  </header>
+  <nav>
+    <div class="container-fluid position-fixed nav-container">
+      <div class="hamburger">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+      <div class="row align-items-center">
+        <div class="col-12 col-md-5 d-flex justify-content-center align-items-center pt-md-0 pb-md-0">
+          <a href="index.html" class="d-flex justify-content-center" ><img src="img/logo.png" class="nav-icon pt-md-0"></a>
+        </div>
+        <div class="col col-md-4 d-flex align-items-center fw-bold">
+          <ul class="nav-links mb-0">
+            <a href="about.html"><li>About</li></a>
+            <a href=""><li>Program</li></a>
+            <a href=""><li>Join</li></a>
+            <a href=""><li>Sponsor</li></a>
+            <a href=""><li>News</li></a>
+          </ul>
+        </div>
+        <div class="col-md-3 align-items-center justify-content-start d-none d-md-flex ps-5">
+          <p class="p-1 m-1 campaign-text">CC Campaign</p>
+        </div>
+      </div>
+    </div>
+  </nav>
+  <section>
+    <div class="container-fluid p-0 intro-bg-layer">
+      <div class="container-fluid" id="intro">
+        <div class="row justify-content-center pt-md-5 mt-md-5">
+          <div class="col-12 col-md-8 p-0 d-flex pt-md-5 flex-column align-items-center">
+            <p class="meeting-count text-center">GET READY FOR SUMMIT 2023!</p>
+            <p class="fs-5 mb-1 intro-hello pt-4 text-center">"GET READY TO CHANGE YOUR LIVE!"</p>
```

Comment:
Just as I stated in the headline section of the home page, kindly ensure to use the right header tag to obey the header hierarchy rule😉.